client/cz-registry.html

```diff
@@ -55,7 +55,10 @@
           }.bind(this));
         }
 
-        return this.registeredSources[type].then(source => source.registerQuery(query, callback));
+        return this.registeredSources[type].then(source => {
+          source.registerQuery(query, callback);
+          return source;
```

Comment:
This gets used again in https://github.com/shans/chromez/issues/77 where the dash asks the issue source to give its interpretation of a username from the user config as it differs between Crbug and Github.